modules/ROOT/pages/authentication-authorization/sso-integration.adoc

```diff
@@ -253,6 +253,109 @@ dbms.security.oidc.mysso.claims.groups=roles
 +
 See xref:authentication-authorization/sso-integration.adoc#auth-sso-map-idp-roles[Map the Identity Provider Groups to the Neo4j Roles].
 
+
+[role=label--new-5.24]
+[[auth-sso-auth-objects]]
+=== Use Auth Objects to configure SSO at the user level
+xref:authentication-authorization/auth-objects.adoc[Auth Objects] can be used to determine which users can authenticate and authorize using the configured providers.
+
+[NOTE]
+====
+You need to set the `dbms.security.require_local_user` configuration setting to `true` to use Auth Objects. This means that a user with a matching Auth Object *must* exist in order to be able to authenticate and authorize.
+====
+
+The following examples show how you can use Cypher to do this.
+
+
+.Create a user with an Auth Object who can authenticate and authorize using `oidc-mysso`.
+======
+----
+CREATE USER jake
+SET AUTH 'oidc-mysso' {SET ID 'jakesUniqueMySso1Id'} // this must match the claim that you configured via dbms.security.oidc.mysso.claims.username
```

Comment:
is `this` the `oidc-mysso` or `jakesUniqueMySso1Id`? I'm guessing the second but it is not completely clear 
```suggestion
SET AUTH 'oidc-mysso' {SET ID 'jakesUniqueMySso1Id'} // the id must match the claim that you configured via dbms.security.oidc.mysso.claims.username
```
or 
```suggestion
SET AUTH 'oidc-mysso' {SET ID 'jakesUniqueMySso1Id'} // the provider must match the claim that you configured via dbms.security.oidc.mysso.claims.username
```
would help clarifying I think